demo/client/build.mjs

```diff
@@ -1,6 +1,6 @@
 import Esbuild from "esbuild";
 import Path from "path";
-import { plugin as extractClientComponents } from "../../packages/extract-client-components/esbuild-plugin.mjs";
+import { plugin as extractClientComponents } from "@ml-in-barcelona/esbuild-plugin";
```

Comment:
ml-in-barcelona¿ server-reason-react/esbuild-plugin no?

Reply:
npm does not allow it, as the package name cannot contain /. I can make call it ml-in-barcelona/server-reason-react-esbuild-plugin